README.md

```diff
@@ -1,11 +1,13 @@
-# Ignite UI Components for React
+# Ignite UI Wrappers for React
 
 [![Build Status](https://travis-ci.org/IgniteUI/igniteui-react.svg?branch=master)](https://travis-ci.org/IgniteUI/igniteui-react?branch=master)
 [![Coverage Status](https://coveralls.io/repos/github/IgniteUI/igniteui-react/badge.svg?branch=master)](https://coveralls.io/github/IgniteUI/igniteui-react?branch=master)
 [![npm version](https://badge.fury.io/js/igniteui-react.svg)](https://badge.fury.io/js/igniteui-react)
 
 Use the declarations available in `igniteui-react.js` (or `igniteui-react.min.js`) to use [Ignite UI](http://igniteui.com) controls as [React](https://facebook.github.io/react/) components. [Work with the running samples here](https://igniteui.github.io/igniteui-react/).
 
+**IMPORTANT** The repository has been renamed from `igniteui-react` to `igniteui-react-wrappers`. This is done to align it to our naming convention explained [here](https://www.infragistics.com/community/blogs/b/infragistics/posts/ignite-ui-github-repo-name-changes). For now the package name will stay with the `igniteui-react` name.
```

Comment:
Add information about the existence of Ignite UI for React, as this is the reason why we rename the repo now.

Reply:
@kdinev Let me know what you think.

Reply:
@gmurray81 Let me know if you want to add some more information regarding the Ignite UI for React product.